package.json

```diff
@@ -40,5 +40,14 @@
     "/bindings/node/",
     "/binding.gyp",
     "/grammar.js"
+  ],
+  "tree-sitter": [
+    {
+      "scope": "source.yaml",
+      "file-types": [
+        "yaml",
+        "yml"
+      ]
```

Comment:
It'd be better that we could align with the [`linguist`](https://github.com/github-linguist/linguist/blob/v7.26.0/lib/linguist/languages.yml#L7878-L7887) data.